### PR TITLE
coverage: Cover ost_utils and ovirtlib

### PR DESCRIPTION
**Pull request by acernek**

### CODE REVIEW

File: lagofy.sh [hunk 1]
```diff
@@ -367,8 +367,9 @@ _ost_run_tc () {
     local res=0
     local testcase=${@/#/$PWD/}
     local junitxml_file="${OST_REPO_ROOT}/exported-artifacts/junit.xml"
+    local coverage_file="${OST_REPO_ROOT}/exported-artifacts/ost_coverage/${SUITE}.$(git rev-parse --short HEAD).coverage"
     source "${OST_REPO_ROOT}/.tox/deps/bin/activate"
-    PYTHONPATH="${PYTHONPATH}:${OST_REPO_ROOT}:${OST_REPO_ROOT}/${SUITE}" ${PYTHON} -u -B -m pytest \
+    PYTHONPATH="${PYTHONPATH}:${OST_REPO_ROOT}:${OST_REPO_ROOT}/${SUITE}" ${PYTHON} -u -B -m coverage run --source=ost_utils,network-suite-master/ovirtlib --data-file=${coverage_file} -m pytest \
```
Comment (erav) [inline lagofy.sh:372]: there is a soft link in ost_utils to network-suite-master/ovirtlib. will specifying ost_utils be sufficient then?

Comment (acernek) [reply lagofy.sh:372]: No, It doesn't seem to suffice.

File: lagofy.sh [hunk 2]
```diff
@@ -382,6 +383,7 @@ _ost_run_tc () {
         xmllint --format ${junitxml_file}
         ./common/scripts/parse_junitxml.py ${junitxml_file} "${OST_REPO_ROOT}/exported-artifacts/result.txt"
     }
+    PYTHONPATH="${PYTHONPATH}:${OST_REPO_ROOT}:${OST_REPO_ROOT}/${SUITE}" ${PYTHON} -u -B -m coverage html -q -d "${coverage_file}-html" --data-file=${coverage_file}
```
Comment (tinez) [inline lagofy.sh:386]: Let's have the html output in exported artifacts too

**Review by tinez — CHANGES_REQUESTED** [review]

File: lagofy.sh (hunk 2 above)
Comment (acernek) [reply lagofy.sh:386]: wdym?

**Review by tinez — APPROVED** [review]

**Pull request merged**